templates/base.html

```diff
@@ -54,6 +54,27 @@
               anchor.classList.add('is-active');
             }
           });
+
+          // Enable Intercom
+          window.intercomSettings = {
+            app_id: "s1rasgdr"
+          };
+          var use_intercom = false;
+          document.querySelectorAll('.p-sidenav__body ul a').forEach(function(a) {
```

Comment:
Rather than working outside in and looping through each `a` you could target only the active `a` and work inside out.

```
  var activeNav = document.querySelector('.p-sidenav__body ul a.is-active');
  if (activeNav) {
    var ul = activeNav.closest("nav > ul");
    if (ul && ul.previousSibling.textContent === "Publishing") {
      use_intercom = true;
    }
  }
```

Reply:
Fixed as 5dbe3ac . Thanks!